openweathermap-ios/.swiftlint.yml

```diff
@@ -0,0 +1,41 @@
+whitelist_rules:
+  - closure_spacing
+  - colon
+  - empty_enum_arguments
+  - fatal_error_message
+  - force_cast
+  - force_try
+  - force_unwrapping
+  - implicitly_unwrapped_optional
+  - legacy_cggeometry_functions
+  - legacy_constant
+  - legacy_constructor
+  - legacy_nsgeometry_functions
+  - operator_usage_whitespace
+  - redundant_string_enum_value
+  - redundant_void_return
+  - return_arrow_whitespace
+  - trailing_newline
+  - type_name
+  - unused_closure_parameter
+  - unused_optional_binding
+  - vertical_whitespace
+  - void_return
+  - custom_rules
+  - indentation
+
+excluded:
+  - Carthage
+  - Pods
+
+colon:
+  apply_to_dictionaries: false
+
+indentation: 2
+
+custom_rules:
+  no_objcMembers:
```

Comment:
A question: This rule make sense in a project only Swift? cc: @henrique-morbin-ifood

Reply:
No, it doesn't. And I not sure about the regex config. Is it right?

Reply:
I compare with internals custom rules here in my current project and this works. I imagine that's a good rule for a swift-format to transform the code. 😄 

I believe we can remove this. @MateusDeSousa what do you think about this?

Reply:
Yes, this rule doesn’t make sense, i did removed. Custom of putting it in my projects😬